src/screens/NewsScreen/Search.js

```diff
@@ -106,7 +106,7 @@ const styles = StyleSheet.create({
     paddingTop: 7,
     paddingBottom: 7,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.alto,
+    borderBottomColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.alto` to `COLORS.lightgrey` is consistent with the PR's goal of enhancing the color theme. However, without knowing the exact colors represented by these variables, it's hard to say whether this change is an improvement or not.
```diff
-    borderBottomColor: COLORS.alto,
+    borderBottomColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->